hub/static/js/all.js

```diff
@@ -1,8 +1,11 @@
-import $ from '../jquery/jquery.esm.js'
-import { Chart, registerables } from '../chartjs/chart.esm.js'
+import $ from 'jquery/dist/jquery.slim'
+//import { Chart, registerables } from '../../../vendor/chartjs/js/chart.esm.js'
```

Comment:
These were only for my history of what I'd changed/testing - don't need to keep these in :) Similar in explore.